### DISCUSSION TITLE
Location info on the main page

### DISCUSSION THREAD

**Discussion by blogdemoi** [mcguirepr89/BirdNET-Pi#525] [Ideas]
Hi,

i think a useful improvement would be the possibility to add a location info on the public pages of Birdnet-Pis. Showing (or not) the place where are located the Birdnet-Pi instances.

What do you think? I just listened to a few birds on https://berowra.birdnetpi.com/ but I have no idea where the recordings are made (in Australia I guess :-) ).

**Comment by mcguirepr89** [comment]
Hi, @blogdemoi -- I would like to leave this up to the site owner. With that said, however, I only issue locatively semantic subdomains, so each subdomain _is_ the location. I need to Google them sometimes to remember where Berowra is, for example, but each subdomain contains the location right there in the URL. 

Also, you can get the general location from the Latitude and Longitude that are reported in the log that any visitor to a public installation can view. The view below is from my installation:
![Screenshot from 2022-09-01 07-39-14](https://user-images.githubusercontent.com/60325264/187905232-cdaf4db7-f48e-4dc3-a339-2fdfe2adbe85.png)

Another option is to include a link to one's BirdWeather station, as those easily inform folks about location: [Berowra's BirdWeather Station](https://app.birdweather.com/stations/200)

If others feel strongly about this, please let me know. I would also like to hear from folks with public installations since their's are the installations that would receive changes.

Thank you for your thoughtful comment -- we'll see what others have to say as well~~

**Comment by Svardsten53** [comment]
In my modification and translation into Swedish, I have included a function where you can specify where the installation is located and this is added to the header. I have also included the country's flag. With these changes, the page becomes more personal and visitors can see where in the world it is located. There are several of us in Sweden who use this function.

[https://tyreso.birdnetpi.com/](url)

**Reply by blogdemoi** [reply]
Thank you, I am currently listening to your birds. Would you mind sharing your modifications? :-).

**Comment by lloydbayley** [comment]
Sounds like a reasonable idea. I would have mine displaying: Berowra, NSW, Australia. for example.
